src/actors/symbolication.rs

```diff
@@ -680,7 +680,20 @@ fn symbolicate_frame(
             }
         };
 
+        // There are a few languages that we should always be able to demangle. Only complain about
+        // those but silently ignore the rest.
         let lang = line_info.language();
+        let should_demangle = matches!(lang, Language::Cpp | Language::Rust | Language::Swift);
+        let demangled_opt = line_info.function_name().demangle(DEMANGLE_OPTIONS);
+        if should_demangle && demangled_opt.is_none() {
+            sentry::with_scope(
+                |scope| scope.set_extra("identifier", line_info.function_name().to_string().into()),
+                || {
+                    let message = format!("Failed to demangle {} identifier", lang);
+                    sentry::capture_message(&message, sentry::Level::Error);
```

Comment:
Do we have no capturing of error logs set up?

Reply:
In symbolicator it's the other way around. Rather than logging unstructured logs, we mostly capture `fail` or errors. In this case, a plain message is enough. 

Now that you're mentioning it, I'm wondering if we should rather use `log::something!` though.

Reply:
it's fine. if that's the pattern (seems to be) then so be it. it's def inconsistent with relay though